stable/elasticsearch/templates/data-types-configmaps.yaml

```diff
@@ -0,0 +1,170 @@
+{{- $elasticsearchFullName :=  include "elasticsearch.fullname" . -}}
```

Comment:
Why are you doing this?
Why don't we use the actual values instead of adding an extra layer of complexity?
It's just another thing people need to lookup to work out what you are doing.

Reply:
From what I can see you are adding quantifiers onto the values in your range function.
I don't see the point of defining a value that is based on another predefined variable which exists.
Why don't we just use the pre-existing variable.

Reply:
I am afraid I am not sure I follow you: could you please give me a specific example so that I can have a better understanding of the problem you are pointing out?
I am not sure if it relates to the example I've shown below, however I could also modify all files so that they refer to root object (saved from outer scope) instead of saving multiple variables. Is this going in the direction you'd like to see?

Reply:
You are defining new variables for this template based on existing variables.
You can just use the original variables. You are not saving much by redefining them.